include/OsqpEigen/Settings.hpp

```diff
@@ -111,15 +111,15 @@ namespace OsqpEigen
 
         /**
          * Set the primal infeasibility tolerance.
-         * @param primalInfeasibilityTollerance tolerace of the primal variables.
+         * @param primalInfeasibilityTolerance tolerace of the primal variables.
          */
-        void setPrimalInfeasibilityTollerance(const double primalInfeasibilityTollerance);
+        void setPrimalInfeasibilityTolerance(const double primalInfeasibilityTolerance);
 
         /**
          * Set the dual infeasibility tolerance.
-         * @param dualInfeasibilityTollerance tolerace of the dual variables.
+         * @param dualInfeasibilityTolerance tolerace of the dual variables.
          */
-        void setDualInfeasibilityTollerance(const double dualInfeasibilityTollerance);
+        void setDualInfeasibilityTolerance(const double dualInfeasibilityTolerance);
```

Comment:
Since this modification will break the `api` could you please deprecate the old methods?

For instance, you can modify the `Setting.hpp` file 

```cpp
/**
 * Set the dual infeasibility tolerance.
 * @param dualInfeasibilityTollerance tolerace of the dual variables.
 */
[[deprecated("Use setDualInfeasibilityTolerance() instead.")]]
void setDualInfeasibilityTollerance(const double dualInfeasibilityTollerance);


/**
 * Set the dual infeasibility tolerance.
 * @param dualInfeasibilityTolerance tolerance of the dual variables.
 */
void setDualInfeasibilityTolerance(const double dualInfeasibilityTolerance);
```

and in the `Setting.cpp` you can have
```cpp
void OsqpEigen::Settings::setDualInfeasibilityTollerance(const double dualInfeasibilityTollerance)
{
    this->setPrimalInfeasibilityTolerance(dualInfeasibilityTollerance);
}

void OsqpEigen::Settings::setDualInfeasibilityTolerance(const double dualInfeasibilityTolerance)
{
    m_settings->eps_dual_inf = (c_float)dualInfeasibilityTolerance;
}
```

Reply:
Thank you @GiulioRomualdi - I've just added commits to add this